project_task_additional_attributes/models/project_task.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    account_manager = fields.Many2one("res.users")
+    view_id = fields.Char(string="View ID", help="List of numeric values")
```

Comment:
```suggestion
    view_id_list = fields.Char(string="View ID", help="List of numeric values")
```
To avoid the confusion of taking the field as many2one.